.github/workflows/build-docs.yml

```
@@ -45,34 +50,33 @@ jobs:
           path: .cache
       - name: Verify README
-        run: diff --brief docs/index.md ./README.md
+        run: python ./scripts/docs.py verify-readme
       - name: Build Docs
-        run: python -m mkdocs build
+        run: python ./scripts/docs.py build
       - uses: actions/upload-artifact@v4
         with:
           name: docs-site
           path: ./site/**
+          include-hidden-files: true
```

actionlint

input "include-hidden-files" is not defined in action "actions/upload-artifact@v4". available inputs are "compression-level", "if-no-files-found", "name", "overwrite", "path", "retention-days"

(action)